metric/emitter/newrelic.go

```diff
@@ -85,3 +108,82 @@ func (emitter *NewRelicEmitter) Emit(logger lager.Logger, event metric.Event) {
 
 	resp.Body.Close()
 }
+
+func (emitter *NewRelicEmitter) Emit(logger lager.Logger, event metric.Event) {
+	payload := make(fullPayload, 0)
+
+	switch event.Name {
+
+	// These are the simple ones that only need a small name transformation
+	case "build started":
```

Comment:
small nit, but rather than `fallthrough` this could just list the values in one `case`. i always forget that `fallthrough` even exists :P